frontend/models/product.ts

```diff
@@ -90,10 +100,46 @@ function getVariants(shopifyProduct: NonNullable<FindProductQuery['product']>) {
          specifications: variant.specifications?.value ?? null,
          warranty: variant.warranty?.value ?? null,
          crossSellVariants: getCrossSellVariants(variant),
+         disableWhenOOS: variant.disableWhenOOS?.value === 'true',
       };
    });
 }
 
+function getImages(
+   shopifyImages: NonNullable<FindProductQuery['product']>['images'],
+   variants: ReturnType<typeof getVariants>
+) {
+   return shopifyImages.nodes.filter(
+      (image) =>
+         !variants.find(
```

Comment:
I might be misreading this boolean expression, but it seems like this will exclude any images with altText, which doesn't seem like what we want.

Reply:
I rewrote the condition to make it more readable.

We are keeping `altText = null` meaning the explicitly generic images
We are filtering out images with altText matching disabled variants
We still keep images with an altText not matching any variant skus - since these might be seo based texts inserted in generic images

Reply:
Thank you for the explanation and the change!